components/31-molecules/map/map.twig

```diff
@@ -1,16 +1,25 @@
 <div id="map" class="map"></div>
-<script src="https://cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@master/en/v5.2.0/build/ol.js"></script>
-<script type="text/javascript">
-  var map = new ol.Map({
-    target: 'map',
-    layers: [
-      new ol.layer.Tile({
-        source: new ol.source.OSM()
-      })
-    ],
-    view: new ol.View({
-      center: ol.proj.fromLonLat([37.41, 8.82]),
-      zoom: 4
-    })
-  });
-</script>
+
+{% include '@heading' with {
+  'type': 'h2',
+  'heading_text': 'Map example'
+} %}
+
+{% include '@image' with {
+  src: '/styleguide/img/stock/map1.png' | path,
+  modifier: 'gentinfo-logo',
+  alt_text: 'Gent info',
```

Comment:
I suppose this is not the Gentinfo logo? 🤨

Reply:
Fixed 😇